aten/src/ATen/native/native_functions.yaml

```diff
@@ -7047,11 +7047,13 @@
 - func: _scaled_mm(Tensor self, Tensor mat2, Tensor scale_a, Tensor scale_b, Tensor? bias=None, Tensor? scale_result=None, ScalarType? out_dtype=None, bool use_fast_accum=False) -> Tensor
   variants: function
   dispatch:
+    CPU: _scaled_mm_cpu
```

Comment:
this thing should also work even mkldnn is not compiled

Reply:
Added a fallback path.